src/OrchardCore.Modules/OrchardCore.Resources/ResourceManagementOptionsConfiguration.cs

```diff
@@ -102,9 +102,9 @@ ResourceManifest BuildManifest()
             manifest
                 .DefineScript("jQuery-ui-i18n")
                 .SetDependencies("jQuery-ui")
-                .SetUrl("~/OrchardCore.Resources/Scripts/jquery-ui-i18n.min.js", "~/OrchardCore.Resources/Scripts/jquery-ui-i18n.js")
```

Comment:
nope this script is present. don't understand why you would remove it.

ok with the other two.

Reply:
Let me check why I did that